scripts/stunnel.conf

```diff
@@ -0,0 +1,33 @@
+# This configurationfile is to use stunnel as a client.
+#
+# The global settings
+#
+
+# Certificate Authority file
+CApath = /etc/ssl/certs/
+
+# Your client certificate in PEM format.
+cert = /tmp/sw444v1912.pem
+
+# Where the private key is kept.
+#key = mycert.pem
+
+# Run in client mode? (remote service is using SSL)
+client = yes
+
+# Debug level (0=nothing, 7 is a lot)||
+debug = 7
+
+# Verification level of remote certificate
+verifyChain = yes
+
+# The value of pid below is kept empty on purpose.
+# Otherwise we got permission errors and it works fine like this.
+pid =
+
+# Very handy for debugging purposes
+#foreground = yes
```

Comment:
Kan weg?

Reply:
Kan weg, maar is fijn om als suggestie te hebben voor mensen die op een later tijdstip willen debuggen.